tests/acceptance/TestHelpers/GraphHelper.php

```diff
@@ -1693,29 +1690,12 @@ public static function getPermissionsRoleIdByName(
 	 *
 	 * @throws \Exception
 	 */
-	public static function getPermissionNameByPermissionRoleId(
-		string $permissionsRoleId
-	): string {
-		switch ($permissionsRoleId) {
-			case 'b1e2218d-eef8-4d4c-b82d-0f1a1b48f3b5':
-				return 'Viewer';
-			case 'fb6c3e19-e378-47e5-b277-9732f9de6e21':
-				return 'Editor';
-			case '2d00ce52-1fc2-4dbc-8b95-a73b73395f5a':
-				return 'File Editor';
-			case '1c996275-f1c9-4e71-abdf-a42f6495e960':
-				return 'Uploader';
-			case 'a8d5fe5e-96e3-418d-825b-534dbdf22b99':
-				return 'Space Viewer';
-			case '58c63c02-1d89-4572-916a-870abc5a1b7d':
-				return 'Space Editor';
-			case '312c0871-5ef7-4b3a-85b6-0e4074c64049':
-				return 'Manager';
-			case 'aa97fe03-7980-45ac-9e50-b325749fd7e6':
-				return 'Secure viewer';
-			default:
-				throw new \Exception('Permission role id: ' . $permissionsRoleId . ' not found');
-		}
+	public static function getPermissionNameByPermissionRoleId(string $permissionsRoleId): string {
+		$all_available_roles = array_merge(self::$DEFAULT_PERMISSIONS_ROLES, self::$ADDITIONAL_PERMISSIONS_ROLES);
+		$roleName = array_search($permissionsRoleId, $all_available_roles);
+		return $roleName;
+
+		throw new \Exception('Permission role id: ' . $permissionsRoleId . ' not found');
```

Comment:
the throw statement will never reach.
```suggestion
		$roleName = array_search($permissionsRoleId, $all_available_roles);
		if ($roleName) {
			return $roleName;
		}

		throw new \Exception('Permission role id: ' . $permissionsRoleId . ' not found');
```